index.d.ts

```diff
@@ -0,0 +1,52 @@
+// TypeScript typings for "mocha-testdata@1.1.3".
+
+declare module "mocha-testdata" {
```

Comment:
That's because it's an ambient definition file. The [non-ambient definition](https://github.com/typed-contrib/knockout/blob/master/index.d.ts) explicitly exports everything and declares a non-string module as the namespace.